pyproject.toml

```diff
@@ -18,6 +18,7 @@ dependencies = [
     "django-celery-beat>=2.6.0",
     "django-celery-results>=2.5.1",
     "django-crispy-forms>=2.1",
+    "django-environ>=0.11.2",
```

Comment:
When a new dependency is added the lock-files need to be regenerated. I wrote a snippet in the docs here: https://docs.spokanetech.org/CONTRIBUTING/#adding-dependencies

(Which, tbf, I wrote after you started.)

Reply:
Nice. Updated lock files.